Makefile

```diff
@@ -237,3 +238,47 @@ catalog-push: ## Push a catalog image.
 # Run CI steps
 .PHONY: ci
 ci: test
+
+.PHONY: api-docs
+api-docs: crdoc kustomize
+	@{ \
+	set -e ;\
+	TMP_DIR=$$(mktemp -d) ; \
+	$(KUSTOMIZE) build config/crd -o $$TMP_DIR/crd-output.yaml ;\
+	$(CRDOC) --resources $$TMP_DIR/crd-output.yaml --output docs/api.md ;\
+	}
+
+
+CRDOC = $(shell pwd)/bin/crdoc
+.PHONY: crdoc
+crdoc: ## Download crdoc locally if necessary.
+	$(call go-get-tool,$(CRDOC), fybrik.io/crdoc,$(CRDOC_VERSION))
+
+
+.PHONY: kustomize
+kustomize: ## Download kustomize locally if necessary.
+	$(call go-get-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v4,$(KUSTOMIZE_VERSION))
+
+# go-get-tool will 'go get' any package $2 and install it to $1.
+PROJECT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
+define go-get-tool
```

Comment:
I know other projects create this macro to download tools but... could we declare it in a separate shell script? That will allow us to add retry logic.
Or, probably, we can use [bingo](https://github.com/bwplotka/bingo) as Observatorium does

Reply:
Can we not add some retry logic to `go-get-tool` if its needed?

Reply:
From the experience we have with GitHub Actions, it is really possible we will need it.

Reply:
i will quickly add a retry logic.

Reply:
Why is a problem to add it now?

Reply:
Until a few minutes ago, that was my plan. However, I have the feeling that this pr is getting bigger and bigger. So it makes sense from my point of view to deliver this afterwards.

Reply:
Does bingo support retries?

Reply:
@frzifus how about creating a new task to not forget about this?
@pavolloffay I'm not sure but I think it has some kind of "recovery logic". It seems bingo is becoming something more standard. But using bingo was just a suggestion about something that could be interesting to implement. No strong opinion

Reply:
I am not a huge fan of bingo, at least I don't see the benefits of introducing another tooling. The go package version can be pinned directly in go get.

Reply:
I agree, its not to complex to download and install specific versions. i would argue with a retry logic. At the end a for loop with a timeout is a simple and sufficient solution.